meson.build

```diff
@@ -292,8 +292,8 @@ if(legacy_meson)
   gmp_dep = find_library('gmp')
   sigsegv_dep = find_library('sigsegv')
 else
-  gmp_dep = meson.get_compiler('c').find_library('gmp')
-  sigsegv_dep = meson.get_compiler('c').find_library('sigsegv')
+  gmp_dep = meson.get_compiler('c').find_library('gmp', dirs:['/usr/local/lib'])
```

Comment:
AFAICT, this means that /usr/local is no longer in the directory search path. You should make this a list of search paths and at have '/usr/lib' in it.